tests/gtest/aviftest_helpers.cc

```diff
@@ -75,24 +70,11 @@ AvifImagePtr CreateImage(int width, int height, int depth,
 }
 
 void FillImagePlain(avifImage* image, const uint32_t yuva[4]) {
-  avifPixelFormatInfo info;
-  avifGetPixelFormatInfo(image->yuvFormat, &info);
-
-  for (int c = 0; c < 4; c++) {
-    uint8_t* row = (c == AVIF_CHAN_A) ? image->alphaPlane : image->yuvPlanes[c];
-    if (!row) {
-      continue;
-    }
-    const uint32_t row_bytes =
-        (c == AVIF_CHAN_A) ? image->alphaRowBytes : image->yuvRowBytes[c];
-    const uint32_t plane_width =
-        (c == AVIF_CHAN_Y || c == AVIF_CHAN_A)
-            ? image->width
-            : ((image->width + info.chromaShiftX) >> info.chromaShiftX);
-    const uint32_t plane_height =
-        (c == AVIF_CHAN_Y || c == AVIF_CHAN_A)
-            ? image->height
-            : ((image->height + info.chromaShiftY) >> info.chromaShiftY);
+  for (avifChannelIndex c :
+       {AVIF_CHAN_Y, AVIF_CHAN_U, AVIF_CHAN_V, AVIF_CHAN_A}) {
+    const uint32_t plane_width = avifImagePlaneWidth(image, c);
+    const uint32_t plane_height = avifImagePlaneHeight(image, c);
+    uint8_t* row = avifImagePlane(image, c);
```

Comment:
Nit: save `avifImagePlaneRowBytes(image, c)` in a local variable `row_bytes` and use `row_bytes` in line 86.

Make similar changes to the three functions that were modified below.